types/types.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+// ContainerInfo stores information about containers
+type ContainerInfo struct {
+	Name            string            `json:"name"`
+	Pid             int               `json:"pid"`
+	Image           string            `json:"image"`
+	CreatedTime     int64             `json:"created_time"`
+	Labels          map[string]string `json:"labels"`
+	Annotations     map[string]string `json:"annotations"`
+	CrioAnnotations map[string]string `json:"crio_annotations"`
```

Comment:
I added this field in my copy of this struct but i'm not seeing what is supposed to populate the value in the server?

https://github.com/kubernetes-incubator/cri-o/blob/master/server/inspect.go#L68-L79

Is there another patch to cri-o that i'm missing?

Reply:
also note that origin already vendors a version of this client code, as part of vendoring cadvisor:

https://github.com/openshift/origin/blob/master/vendor/github.com/google/cadvisor/container/crio/client.go#L38-L48

So should cadvisor's containerinfo struct also be patched with this new annotation?  should this client code not go into cri-o and it should stay in cadvisor?